v3/internal/assetserver/assetserver_webview.go

```
@@ -12,6 +12,40 @@ import (
 	"github.com/wailsapp/wails/v3/internal/assetserver/webview"
 )
 
+type requestContextResponseWriter struct {
+	webview.ResponseWriter
+	ctx context.Context
+}
+
+func (rw *requestContextResponseWriter) Write(buf []byte) (int, error) {
+	if rw.ctx.Err() != nil {
+		return len(buf), nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 21) but it returns nil

(nilerr)